apps/dashboard/lib/tinybird.ts

```diff
@@ -768,7 +768,7 @@ export const getSemanticCachesDaily = tb.buildPipe({
 });
 
 export const getSemanticCachesHourly = tb.buildPipe({
-  pipe: "get_semantic_caches_hourly__v1",
+  pipe: "get_semantic_caches_hourly__v2",
```

Comment:
is that pipe in tinybird prod?

Reply:
Yeah https://app.tinybird.co/gcp/europe-west3/unkey/pipe/get_semantic_caches_hourly__v2